modelconverter/platforms/rvc4/analyze.py

```
@@ -198,13 +198,13 @@ def _prepare_input_matcher(self) -> dict[str, dict[str, str]]:
 
     def _prepare_raw_inputs(
         self,
-        input_matcher: dict[str, dict[str, str]],
+        input_matcher: dict[int, dict[str, str]],
         type: type = np.uint8,
```

Ruff (0.16.2)

[error] Function argument `type` is shadowing a Python builtin

(A002)